### ISSUE TITLE
Cannot save and load complete tracing records locally

### ISSUE THREAD

**Issue by Howie-Arup** [Arize-ai/phoenix#4673]
Hi, I am using Phoenix locally to trace my LLM calls and save the tracing results. The codes are as below:

```
# Save
directory = 'traces'
trace_id = px.Client().get_trace_dataset().save(directory=directory)

# Load
trace_dataset = px.TraceDataset.load(trace_id, directory=directory)
px.launch_app(trace=trace_dataset)
```

But the loaded record misses the `Input Messages` part which is more friendly for reading.
Original result before saving:
![original result](https://github.com/user-attachments/assets/54d193c9-7861-4084-876e-e3de8db90ce3)
Loaded result that misses the `Input Messages` part:
![loaded result](https://github.com/user-attachments/assets/74287c10-613e-4b4e-a087-b56588e99976)

How to save all the parts of the tracing record? Thanks!

**Comment by mikeldking** [comment]
@Howie-Arup Thanks for reporting! Will take a look

**Comment by RogerHYang** [comment]
@Howie-Arup Thanks for the report. Is this occurring in the latest version Phoenix?

**Comment by Howie-Arup** [comment]
> @Howie-Arup Thanks for the report. Is this occurring in the latest version of Phoenix?

Thanks for reply. I am using the below versions:

```
arize-phoenix             4.32.0                   pypi_0    pypi
arize-phoenix-evals       0.15.1                   pypi_0    pypi
arize-phoenix-otel        0.5.0                    pypi_0    pypi
```

**Comment by RogerHYang** [comment]
We've released a fix in v4.35.2. Please check it out and let us know if you need anything else. Thanks again!
